Tests/test_content.py

```diff
@@ -720,19 +640,17 @@ def run_test_scenario(tests_queue, tests_settings, t, proxy, default_test_timeou
     test_to_version = t.get('toversion', '99.99.99')
 
     if not (LooseVersion(test_from_version) <= LooseVersion(server_numeric_version) <= LooseVersion(test_to_version)):
-        prints_manager.add_print_job(f'\n------ Test {test_message} start ------', print, thread_index,
-                                     include_timestamp=True)
-        warning_message = 'Test {} ignored due to version mismatch (test versions: {}-{})'.format(test_message,
-                                                                                                  test_from_version,
-                                                                                                  test_to_version)
-        prints_manager.add_print_job(warning_message, print_warning, thread_index)
-        prints_manager.add_print_job(f'------ Test {test_message} end ------\n', print, thread_index,
-                                     include_timestamp=True)
+        logging_manager.info(f'------ Test {test_message} start ------')
+
+        warning_message = \
+            f'Test {test_message} ignored due to version mismatch (test versions: {test_from_version}-{test_to_version})'
+        logging_manager.warning(warning_message)
+        logging_manager.info(f'------ Test {test_message} end ------\n')
```

Comment:
Why do we print a test start message and test end message for tests we're skipping? Can't we just print the ignore message?

Reply:
Sometimes you have a point